src/OrbitBase/Logging.cpp

```diff
@@ -14,6 +14,24 @@
 
 static absl::Mutex log_file_mutex(absl::kConstInit);
 std::ofstream log_file;
+const int kLogFileLifetimeMinutes = 60 * 24 * 7;  // one week
+
+void RemoveOldLogFiles(const std::filesystem::path& log_dir) {
+  auto now = std::filesystem::file_time_type::clock::now();
+  for (const auto& log_file_path : std::filesystem::recursive_directory_iterator(log_dir)) {
+    std::chrono::minutes file_lifetime_minutes = std::chrono::duration_cast<std::chrono::minutes>(
+        now - std::filesystem::last_write_time(log_file_path));
```

Comment:
hmm... another alternative would be to parse filename in order to extract the date, and remove files based on name rather than last_modified time. And this also is easily testable since you can create such a file and then remove it.

Reply:
Cool suggestion! It is so easy to test! I changed to extract the time_point from filename. Thanks!